src/frontend/src/pages/ProjectDetailPage/ProjectForm/ProjectEditContainer.tsx

```diff
@@ -117,6 +122,41 @@ const ProjectEditContainer: React.FC<ProjectEditContainerProps> = ({ project, ex
     }
   };
 
+  const onSubmitCreateChangeRequest = async (data: ProjectCreateChangeRequestFormInput) => {
```

Comment:
move this into the form, it can be used for both create and edit i believe kill two birds with one stone